docs/get-started.md

```diff
@@ -1,170 +1,64 @@
-need to ensure that you provide an instance of [react-router].
-
-As such, we suggest that you use our specially designed tech stack for
-this purpose. You can start a brand new project or prototype using the
-following steps.
+Prototyping
+===========
 
+This guide explains how to create prototypes using the Home Office Design System.
 
 ### 1. Set up your repository
 
-You should first set up a blank code repository in the provider of your
-choice. For the purposes of this document I will assume [GitHub] as we
-provide extra CI support through [GitHub Actions].
+Set up a blank code repository in [GitHub](https://github.com/) as
+extra CI support is provided through [GitHub Actions](https://github.com/features/actions).
 
 
 ### 2. Run the following commands locally
 
-(The parts in ALL CAPS should be replaced with something specific to
-you.)
+Using terminal, run the following commands below (the parts in ALL CAPS should be replaced with something specific to
+you):
 
 ```shell
-mkdir YOUR-PROJECT
-cd YOUR-PROJECT
+mkdir YOUR-PROTOTYPE
+cd YOUR-PROTOTYPE
 git init
-git remote add origin git@github.com:YOUR_USER/YOUR-PROJECT.git
-npm init @hods
+git remote add origin git@github.com:YOUR_USER/YOUR-PROTOTYPE.git
+npm init @hods prototype
 ```
 
 
 ### 3. Answer the interactive prompts
 
-Our generator / initialiser will ask you some questions about your new
-project.
+The generator / initialiser will ask you some questions about whether you want to create a new prototype or
+project:
 
-The most important choice is whether to start a new project or
-a prototype:
 - **Prototype**
-  A simple stand-alone application. Use this if you just want to play
-  around with the components or you want to build a prototype prior to
-  starting a full project.
-  (If you already have a project, you might prefer to run
+  Select this option if you want to build a prototype for testing ideas and for research.
+  (If you already have a prototype, you might prefer to run
   `npm run create:app` instead.)
 - **Project**
-  A monorepo designed to handle an entire project including multiple
-  applications, and reusable libraries and components.
+  This option is aimed at developers to set up a project in production using the
+  coded React examples from the Home Office Design System.
 
-If you create a new project you should pull down dependencies at this
-point in order to update your lock-file:
+Once you've selected the prototype option and answered the additional prompts you should pull down the dependencies:
 
 ```shell
 pnpm install
 ```
 
 
-### 4. Commit and push your new project
+### 4. View your prototype locally
+
+Enter the following command:
+
+```shell
+npm run dev
+```
+Once that's completed, open in your browser: http://localhost:8080/
+
+### 5. Commit and push your new prototype
 
 Run the following commands to push up the first version of your new
-project to your remote code repository:
+prototype to your remote code repository:
```

Comment:
I think this should be optional and we should move the `git remote add` line down to here.

Reply:
Done, here [PR #57](https://github.com/UKHomeOffice/hods-poc/pull/57)